content/v2.0/reference/syntax/delete-predicate.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Delete predicate syntax
+list_title: Delete predicate
+description: >
+  The InfluxDB `/delete` endpoint uses a SQL-like predicate syntax to determine
+  what data points to delete.
+menu:
+  v2_0_ref:
+    parent: Syntax
+    name: Delete predicate
+weight: 104
+v2.0/tags: [syntax, delete]
+related:
+  - /v2.0/reference/cli/influx/delete/
+---
+
+The InfluxDB `/delete` endpoint uses a SQL-like predicate syntax to determine
+what data [points](/v2.0/reference/glossary/#point) to delete.
+InfluxDB uses the delete predicate to evaluate each point in the time range
+specified in the delete request.
+Points that evaluate to `true` are deleted.
+Points that evaluate to `false` are preserved.
+
+A delete predicate is comprised of one or more [predicate expressions](/v2.0/reference/glossary/#predicate-expression).
+The left operand of the predicate expression is the column name.
+The right operand is the column value.
+Operands are compared using [comparison operators](#comparison-operators).
+Use [logical operators](#logical-operators) to combine two or more predicate expressions.
+
+##### Example delete predicate
+```sql
+key1="value1" AND key2="value"
+```
+{{% note %}}
+Predicate expressions can use any column or tag except `_time` or `_value`.
+{{% /note %}}
+
+## Logical operators
+Logical operators join two or more predicate expressions.
+
+| Operator | Description                                                                  |
+|:-------- |:-----------                                                                  |
+| `AND`    | Both left and right operands must be `true` for the expression to be `true`. |
+
+## Comparison operators
+Comparison operators compare left and right operands and return `true` or `false`.
+
+| Operator | Description  | Example        | Result |
+|:-------- |:-----------  |:-------:       |:------:|
+| `=`      | Equal to     | `"abc"="abc"`  | `true` |
+| `!=`     | Not equal to | `"abc"!="def"` | `true` |
```

Comment:
I checked and `!=` isn't supported yet. We will add it I'm sure, but I would omit it for now.